components/KatFlips/KatFlips.tsx

```diff
@@ -43,16 +43,16 @@ let observer: MutationObserver
 
 export function KatFlips(props: Props) {
     let [nameFilter, setNameFilter] = useState<string | null>()
+    let [minimumTime, setMinimumTime] = useState<number>(0)
```

Comment:
I dont think the minimumTime is needed as a filter as it already exits as a sorting option.
This also has the benefit that the user is able to "search" for the thing with lower and higher time depending on what they want.

The minProfit filter can stay because I somehow can see people using that despite it not really being needed as well (as it is sorted by profit by default)

Reply:
For profit/time, pets like guardian which take one hour come at the top, but for people who need to AFK for longer than one hour, its helpful

Reply:
Yeah but coulnt these people just use the sorting function? I feel like this is a feature that wont really be used and takes up space..

If you are really sure that people want that I would ask the other main devs or other people using the page what they think about it

Reply:
sure, i'll remove it if you insist

Reply:
done